content/docs/command-reference/repro.md

```diff
@@ -286,3 +286,7 @@ $ dvc dag
       |  count  |
       `---------'
 ```
+
+> Note that using `dvc repro --downstream` without a target will always have a
+> similar effect, where all previous stages are ignored — only if the last stage
+> is changed will it have any effect.
```

Comment:
I guess let's just update the example to insert a 3rd stage and target the 2nd one with --downstream, instead of needing this note. Although the note could be useful in the option desc.

Reply:
We can also extend the `--downstream` opt. with the sentence "If no `targets` are provided, this option is ignored" along with an update in the example.

Reply:
Yes. But it's not exactly ignored. It just doesn't have any effect.

Reply:
Yeah actually, could we replace "this option is ignored" with "this option has no effect"?

Reply:
@sarthakforwet please apply the changes without waiting for me to confirm every action. It's not a sustainable workflow. I can always review them later.